Source/player.cpp

```diff
@@ -3504,8 +3504,6 @@ void CheckNewPath(int pnum)
 				if (object[i]._oBreak == 1) {
 					d = GetDirection(plr[pnum].position.current.x, plr[pnum].position.current.y, object[i]._ox, object[i]._oy);
 					StartAttack(pnum, d);
-				} else {

```

Comment:
What happens if we just remove the entire `ACTION_OPERATE` block? Seems like it only exists for melee attacks (for some weird reason) even though you can be on melee while casting or while firing a bow.

Reply:
well I guess it wouldn't allow you to break barrels fast

Reply:
Oh is breaking barrels an operate instead of an attack?! Interesting! So _that's_ why this was here?

Reply:
well, it performs a check for breaking and barrels are the only breakable object in the game iirc : P